members/_posts/2016-10-04-lewic.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : lewic.png

```

Comment:
I would prefer only one Picture, scaling is done by html. But as I see max width and height should be 384x384 or even 400x400 pixel. This is enough! :-)